eval.py

```diff
@@ -84,76 +122,156 @@ def single_replicate_performances():
     return df_performances
 
 
+# TODO: just want to make sure I understand this
 def compute_cis(series, confidence_level):
-    """
-    Compute confidence intervals given cf level
-    """
     sorted_perfs = series.sort_values()
     lower_index = int(confidence_level/2 * len(sorted_perfs)) - 1
     upper_index = int((1 - confidence_level/2) * len(sorted_perfs)) - 1
     lower = sorted_perfs.iloc[lower_index].round(3)
     upper = sorted_perfs.iloc[upper_index].round(3)
-    mean = round(sorted_perfs.mean(), 3)
+    mean = round(sorted_perfs.mean(),3)
     return lower, mean, upper
 
 
-def create_ci_record(perfs, name):
-    lower, mean, upper = compute_cis(perfs, confidence_level=0.05)
-    record = {"name": name,
+# TODO: what does "perfs" stand for? And is there a reason we're using "name"?
+def create_ci_record(perfs, task):
+    lower, mean, upper = compute_cis(perfs, confidence_level = 0.05)
+    record = {"name": task,
               "lower": lower,
               "mean": mean,
-              "upper": upper,
-              }
+              "upper": upper}
     return record
 
 
-def evaluate(gt_dir, pred_dir, save_dir, table_name):
+def create_map(pkl_path):
     """
-    Pipeline to evaluate localizations. Return miou by pathologies and their confidence intervals
+    Create saliency map of original img size·
     """
+    info = pickle.load(open(pkl_path,'rb'))
+    saliency_map = info['map']
+    img_dims = info['cxr_dims']
+    map_resized = F.interpolate(saliency_map, size=(img_dims[1],img_dims[0]), mode='bilinear', align_corners=False)
+    saliency_map = map_resized.squeeze().squeeze().detach().cpu().numpy()
+    return saliency_map, img_dims
+
 
-    # create save dir if it doesn't exist
-    Path(save_dir).mkdir(exist_ok=True, parents=True)
+def get_hit_rates(gt_path, pred_path):
+    """
+    TODO: make this comment clearer
```

Comment:
Let's discuss, because it looks like this function expects a different path? like a path with a bunch of pkl files. is that right? where do these pkl files come from?